### ISSUE TITLE
HttpServer from commandline ignores languagemodel option

### ISSUE THREAD

**Issue by arcanefoam** [languagetool-org/languagetool#404]
Hello, I am trying to use ngram data on my LT server from the comandline on Windows. I am launching the server like this:

java -cp languagetool-server.jar org.languagetool.server.HTTPServer --port 8081 --langaugemodel "C:\LanguageTool-ngrams"

Where the LanguageTool-ngrams is a folder containing the ngram data (en folder for the english data).

I send one of the test senences to the server:

http://localhost:8081/?language=en&text=Alabama has for of the world's largest stadiums

And I get no errors:
<?xml version="1.0" encoding="UTF-8"?>
<matches software="LanguageTool" version="3.2" buildDate="2015-12-29 10:24">
<language shortname="en" name="English"/>
</matches>

I can verify that ngram data is being used by running the LT gui, in which case all the test senteces result in the expected errors

**Comment by danielnaber** [comment]
You have a typo in `--languagemodel`

**Comment by arcanefoam** [comment]
Hi,yeah, sorry for the typo. With out the type the result is the same, i.e. not working as expected.

**Comment by jambt** [comment]
I second this observation on a linux machine with languagetool 3.3. The server mode of languagetool does not incorporate n-grams.



Steps to reproduce:
1) verify that n-gram support is working:

`
~  languagetool --languagemodel "/opt/n-grams" --language en-US
Expected text language: English (US)
I want to go their.
Working on STDIN...
1.) Line 1, column 14, Rule ID: CONFUSION_RULE
Message: Statistic suggests that 'there' (as in 'Is there an answer?') might be the correct word here, not 'their' (as in 'It’s not their fault.'). Please check.
Suggestion: there
I want to go their. 
             ^^^^^  
Time: 2440ms for 1 sentences (0.4 sentences/sec)`

Step 2: Start the server with `--http`:

`
~  languagetool --languagemodel "/opt/n-grams" --language en-US --http
WARNING: running in HTTP mode, consider using org.languagetool.server.HTTPSServer for encrypted connections
Setting up thread pool with 10 threads
Starting LanguageTool 3.3 (build date: 2016-03-28 09:17) server on http://localhost:8081...
Server started
`

Step 3a: request the sentence 

`
~  curl http://localhost:8081 -d "text=I want to go their." -d language=en-US
<?xml version="1.0" encoding="UTF-8"?>
<matches software="LanguageTool" version="3.3" buildDate="2016-03-28 09:17">
<language shortname="en-US" name="English (US)"/>
</matches>
`

Step 3b: Log from the server:

`2016-05-10 12:07:28 Check done: 19 chars, en-US, null, handlers:1, queue:0, 0 matches, 5ms, agent:-, sent`


Expected result: See the error from Step 1.a

**Comment by danielnaber** [comment]
The wiki documentation is misleading, I'll fix that. Please call `java -cp languagetool-server.jar org.languagetool.server.HTTPServer --help` to see the usage -> you'll need to have a `languageModel` option in the properties file, not as a parameter.

**Comment by jambt** [comment]
I confirm: passing `--config ~/.languagetool.cfg` and inserting the line `languageModel=/opt/n-grams` solves the problem.

This is very misleading, but fixes the problem for me. Thank you for your help.

**Comment by arcanefoam** [comment]
I came up with this buf when trying to run LT from TexStudio. Having this setting in a properties file will probably make it harder to integrate to other tools. If it were a comandline option, then is just an additional paramter to pass in the options of TexStudio (aka the other tool). 

Added the setting to the configuration file and now it works as expected. On a side note. The .languagetool.cfg file generate (appratenly) by the LT GUI has a ngramDir setting. Would it be preferable to merge this two options into one? I had tried running the server with that cfg file before, but I am guesing the server does not understand the ngramDir option.
